firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -117,6 +118,16 @@
           .setRegistrationStatus(PersistedFid.RegistrationStatus.UNREGISTERED)
           .build();
 
+  private static final PersistedFidEntry INVALID_FID_ENTRY =
+      PersistedFidEntry.builder()
+          .setFirebaseInstallationId(INVALID_TEST_FID)
+          .setAuthToken("")
+          .setRefreshToken("")
+          .setTokenCreationEpochInSecs(TEST_CREATION_TIMESTAMP_1)
+          .setExpiresInSecs(0)
```

Comment:
Assuming that this is an object (or a proto), can we create a class (or proto) for Auth-Token that has the fields "token", "creationTimestamp", "expirationTimestamp" or "expiresIn", so that the fields that belong together are part of the same object?

Reply:
I can change but IMO dont see much benefit. This object is used to represent persisted fid & metadata - simpler the class, easier it is to read and write. 

Objects that are exposed to users have been separated out into a class (or proto) for Auth-Token & a class for firebase installation. This addresses your comment & is useful to segregate the details to the clients.